backend/app/services/schwab_csv.py

```
@@ -0,0 +1,271 @@
+    """Parse a Schwab option symbol into (ticker, expiration, strike, putCall).
+
+    Returns None if the symbol is empty, malformed, or not an option (e.g.
+    plain equity tickers like "AAPL").
+    """
+    if not symbol:
+        return None
+
+    human = _HUMAN_SYMBOL_RE.match(symbol)
+    if human:
+        try:
+            exp = datetime.strptime(human.group("expiration"), "%m/%d/%Y").strftime("%Y-%m-%d")
```

Ruff (0.15.12)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)